packages/e2e-tests/scripts/record-playwright.ts

```diff
@@ -7,9 +9,21 @@ import config, { BrowserName } from "../config";
 
 export async function recordPlaywright(
   browserName: BrowserName,
-  script: (page: Page) => Promise<void>
+  script: (page: Page, expect?: typeof expectFunction) => Promise<void>
 ) {
-  const browserEntry = playwright[browserName];
+  // Dynamically import `@playwright/test`, for two reasons:
+  // - FF only works with PT <=1.19
+  // - PT errors if it gets imported more than once
+  // WARNING: `playwright-test1_19` is not installed by default!
+  // Installing it clobbers the `./node_modules/.bin/playwright` symlink,
+  // which then breaks the ability to _run_ some of our E2E tests.
+  // Right now I'm recording FF examples and then uninstalling that version.
+  // TODO Figure out how to get these to co-exist so we can re-record more FF examples.
+  const playwrightTestPackage =
+    browserName === "chromium" ? "@playwright/test" : "playwright-test1_19";
```

Comment:
Hey @markerikson

What is `playwright-test1_19` ? Did you forget to commit an update to `package.json`/`yarn.lock` ?

![Screenshot 2024-01-23 at 9 08 02 AM](https://github.com/replayio/devtools/assets/29597/b8c76feb-7b32-4765-bac6-5adf4c301e48)

Reply:
I should have read the inline comment more thoroughly before commenting. I see it looks like this was intentional because two Playwrights can't co-exist.

I guess I'll try to remove this entirely (since we don't need to re-record Gecko recordings).